airflow/contrib/hooks/opsgenie_alert_hook.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import json
+
+from airflow.hooks.http_hook import HttpHook
+
+
+class OpsgenieAlertHook(HttpHook):
+    """
+    This hook allows you to post alerts to Opsgenie.
+    Accepts a connection that has an Opsgenie API key as the connection's password.
+    Each Opsgenie API key can be pre-configured to a team integration.
+    You can override these defaults in this hook.
+
+    :param http_conn_id: Http connection ID with host as "https://api.opsgenie.com/"
+                      and Opsgenie API key as the connection's password
+                      (e.g. "eb243592-faa2-4ba2-a551q-1afdf565c889")
+    :type http_conn_id: str
+    :param payload: Opsgenie API Create Alert payload values
+                    See https://docs.opsgenie.com/docs/alert-api#section-create-alert
+    :type payload: dict
+    :param proxy: Proxy to use to make the Opsgenie Alert API call
+    :type proxy: str
+    """
+    def __init__(self,
+                 http_conn_id,
+                 payload={},
+                 proxy=None,
+                 *args,
+                 **kwargs
+                 ):
+        super(OpsgenieAlertHook, self).__init__(*args, **kwargs)
```

Comment:
Maybe 
```suggestion
        super(OpsgenieAlertHook, self).__init__(http_conn_id=opsgenie_conn_id, *args, **kwargs)
```
and remove L51 
https://github.com/apache/airflow/pull/4895#discussion_r268206731

Reply:
Thanks for the heads up, fixed in latest push.